SpatialGDK/Source/SpatialGDK/Private/Interop/Connection/SpatialTraceUniqueId.cpp

```diff
@@ -14,3 +14,9 @@ EventTraceUniqueId EventTraceUniqueId::GenerateForRPC(Worker_EntityId Entity, ui
 	uint32 ComputedHash = HashCombine(HashCombine(GetTypeHash(static_cast<int64>(Entity)), GetTypeHash(Type)), GetTypeHash(Id));
 	return EventTraceUniqueId(ComputedHash);
 }
+
+EventTraceUniqueId EventTraceUniqueId::GenerateForProperty(Worker_EntityId Entity, const GDK_PROPERTY(Property) * Property)
```

Comment:
```suggestion
EventTraceUniqueId EventTraceUniqueId::GenerateForProperty(Worker_EntityId Entity, const GDK_PROPERTY(Property)* Property)
```
What's that space doing there

Reply:
Clang formatting confused with the macro, I agree it's chaos a space before and after *

Reply:
We gonna leave as is since clang is being mean?